src/services/user.service.js

```diff
@@ -0,0 +1,67 @@
+import models from '../models';
+
+const { Users } = models;
+
+/**
+ * @exports
+ * @class UserService
+ */
+class UserService {
+  /**
+ * create new user
+ * @static
+ * @param {object} newUser
+ * @memberof UserService
+ * @returns {object} data
+ */
+  static addUser(newUser) {
+    return Users.create(newUser);
+  }
+
+  /**
+ * find user by email
+ * @static
+ * @param {object} userEmail
+ * @memberof UserService
+ * @returns {object} data
+ */
+  static findByEmail(userEmail) {
+    return Users.findOne({
+      where: {
+        email: userEmail
+      },
+      logging: false
+    });
+  }
+
+  /**
+   *
+   *
+   * @static
+   * @param {item} user
+   * @param {value} token
+   * @returns {updated} @memberof UserService
+   */
+  static updateToken(user, token) {
```

Comment:
This can be a generic update method instead so it can be used for updating token and other fields.